package.json

```diff
@@ -54,7 +54,7 @@
     "vector-tile": "^1.3.0",
     "velocity-react": "^1.3.3",
     "whatwg-fetch": "^2.0.3",
-    "globalfishingwatch-convert": "git+https://github.com/GlobalFishingWatch/map-convert.git"
+    "globalfishingwatch-convert": "git@github.com:GlobalFishingWatch/map-convert.git"
```

Comment:
@nerik why did you changed the protocol?

Reply:
yeah, I don't think we should do this because we run this build process in some GCP machines that are not set up for ssh access.